### PR TITLE
Send CMD as notifications

### PR DESCRIPTION
**Pull request by jsalus**

### CODE REVIEW

**Comment by jsalus** [comment]
@fvacek

File: libshviotqt/src/node/shvnode.cpp [hunk 1]
```diff
@@ -263,6 +263,14 @@ chainpack::RpcValue ShvNode::processRpcRequest(const chainpack::RpcRequest &rq)
 		rootNode()->emitLogUserCommand(e);
 	}
 
+	if(mm_access_level == cp::MetaMethod::AccessLevel::Command) {
+		cp::RpcSignal sig;
+		sig.setMethod(cp::Rpc::SIG_VAL_CHANGED);
```
Comment (fvacek) [inline libshviotqt/src/node/shvnode.cpp:268]: `method + '(' + rq.params().toCpon() + ')'`

Comment (fvacek) [reply libshviotqt/src/node/shvnode.cpp:268]: do params dat DataChange

File: libshviotqt/src/node/shvnode.cpp [hunk 2]
```diff
@@ -263,6 +263,14 @@ chainpack::RpcValue ShvNode::processRpcRequest(const chainpack::RpcRequest &rq)
 		rootNode()->emitLogUserCommand(e);
 	}
 
+	if(mm_access_level == cp::MetaMethod::AccessLevel::Command) {
```
Comment (fvacek) [inline libshviotqt/src/node/shvnode.cpp:266]: cp::MetaMethod::AccessLevel::Write

**Pull request merged**